### PR TITLE
Install and configure webform enccrypt

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Installs Webform Encrypt, Encrypt, Key, and Real AES modules.
Configures the encryption profile and key required.
Deletes the default contact webform during site install.
Adds a .env file to the develop script.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIG-74

### CODE REVIEW

File: ecms_base/ecms_base.install [hunk 1]
```diff
@@ -66,4 +66,9 @@ function ecms_base_install() {
       ->addContentTypeToWorkflow($type->id());
   }
 
+  // Delete the default 'contact' webform.
+  \Drupal::entityTypeManager()
```
Comment (pfrilling) [inline ecms_base/ecms_base.install:70]: Does this remove the 'Contact' node that is created too? Should it?

**Review by pfrilling — APPROVED** [review]
The code looks good. I just had one question about the webform delete.

File: ecms_base/ecms_base.install (hunk 1 above)
Comment (ben-hamelin) [reply ecms_base/ecms_base.install]: Yes, good call. I've added an extra step to the install hook.

**Comment by ben-hamelin** [comment]
@pfrilling ready for review

**Review by pfrilling — APPROVED** [review]
Looks good. 👍

**Pull request merged**